lab_7_llm/main.py

```diff
@@ -41,20 +42,38 @@ def obtain(self) -> None:
         Raises:
             TypeError: In case of downloaded dataset is not pd.DataFrame
         """
+        self._raw_data = load_dataset("OxAISH-AL-LLM/wiki_toxic",
+                                      name=self._hf_name,
```

Comment:
I think it’s better to use a `self._hf_name`, since it already contains `OxAISH-AL-LLM/wiki_toxic`

Reply:
ok!